components/PackageList.tsx

```diff
@@ -8,19 +9,28 @@ type PackageListProps = {
 }
 
 const PackageList: React.FC<PackageListProps> = (props) => {
+    const [packageList, setPackageList] = useState(props.items as { name: string, version: string }[]);
```

Comment:
This `as` should be unnecessary here, as the type is already correctly inferred from `props.items`.